app/web/src/components/ChangesPanelHistory.vue

```diff
@@ -1,5 +1,9 @@
 <template>
-  <ScrollArea ref="mainDivRef" @click="deselectOnClickEmptySpace">
+  <div
+    v-if="featureFlagsStore.MANAGEMENT_FUNCTIONS"
```

Comment:
@wendybujalski I'll pull this down to test the flag in a second. But will this hide the *entire* history if you don't have the flag on? We just want to hide the management func history

Reply:
Nah, I just separated the two options for flag on/off to make it simpler.